convert_checkpoints.py

```diff
@@ -403,16 +404,70 @@ def merge_weights(
   print(f"Export outputs takes {end - start} seconds")
 
 
+def convert_hf_gemma_weights(
+    input_ckpt_dir: epath.Path, output_ckpt_dir: epath.Path
+):
+  """Convert gemma weights from Huggingface to be compatible with JetStream
+  1. Map attention weights to new names.
+  2. Split qkv fusion.
+  """
+  ckpt_file = list(input_ckpt_dir.glob("*.ckpt"))
+  assert len(ckpt_file) == 1
+  ckpt_file = ckpt_file[0]
+  state_dict = torch.load(ckpt_file, map_location=torch.device("cpu"))[
+      "model_state_dict"
+  ]
+  model_config = json.loads((input_ckpt_dir / "config.json").read_text())
```

Comment:
you can change `json.load` with `eval` to skip the need to change `'` to `"` in the instructions.

Reply:
If using `eval`, we need to change `false` to `False` in the `config.json`

Reply:
or let's read the config from https://github.com/google/jetstream-pytorch/blob/main/jetstream_pt/third_party/gemma/config.py#L81 using --size argument?

Reply:
I think it's better to submit a PR to huggingface to correct the error. It should be an valid JSON. Not all Gemma ckpt on HF have this issue.

Reply:
I opened a PR at https://huggingface.co/google/gemma-7b-pytorch/discussions/2